### PR TITLE
feat(frontend): add footer links

### PR DESCRIPTION
**Pull request by UnbearableBear**
C'est tout bête

### CODE REVIEW

File: packages/code-du-travail-frontend/src/layout/Footer.js [hunk 1]
```diff
@@ -148,6 +148,55 @@ const Footer = () => (
...
       </NavSection>
+      <GovernmentSection>
+        <StyledGovLink
+          href={"https://travail-emploi.gouv.fr"}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          travail-emploi.gouv.fr
+        </StyledGovLink>
+        <Separator>|</Separator>
```
Comment (lionelB) [inline packages/code-du-travail-frontend/src/layout/Footer.js:159]: I would add an aria-hidden  / user-select: none;to Separator element

**Review by lionelB — CHANGES_REQUESTED** [review]
Just a small issue with `<Separator/>` element

File: packages/code-du-travail-frontend/src/layout/Footer.js (hunk 1 above)
Comment (revolunet) [reply packages/code-du-travail-frontend/src/layout/Footer.js:159]: added

**Comment by lionelB** [comment]
need a rebase

**Review by lionelB — APPROVED** [review]

**Comment by UnbearableBear** [comment]
duplicate pr

**Pull request closed without merging**